### PR TITLE
Hibernate util

### PR DESCRIPTION
**Pull request by sabiutayo**

### CODE REVIEW

File: src/main/java/sample/Controller.java [hunk 1]
```diff
@@ -1,4 +1,9 @@
 package sample;
 
 public class Controller {
+
```
Comment (stefan2212) [inline src/main/java/sample/Controller.java:4]: Why so many white spaces?

File: src/main/java/sample/HibernateUtil.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            settings.put(Environment.USER, "root");
+            settings.put(Environment.PASS, "root");
+            settings.put(Environment.DIALECT, "org.hibernate.dialect.MySQL5Dialect");
+            settings.put(Environment.SHOW_SQL, "true");
+            settings.put(Environment.CURRENT_SESSION_CONTEXT_CLASS, "thread");
+            settings.put(Environment.HBM2DDL_AUTO, "update");
+            configuration.setProperties(settings);
+
+
+           // configuration.addAnnotatedClass(User.class);
```
Comment (stefan2212) [inline src/main/java/sample/HibernateUtil.java]: Delete commented code

**Review by stefan2212 — CHANGES_REQUESTED** [review]

**Review by stefan2212 — APPROVED** [review]
LGTM!

**Review by Bossy12 — APPROVED** [review]
All good!!

**Pull request merged**